Home page SEO improvements

nicktiberi opened the pull request:
Fixes #720.

- meta description added to home page
- "read more" link is now more descriptive

awead commented:
@cam156 well, that's interesting.

cam156 commented:
@awead That is interesting.  I'm guessing coveralls uses the rpsec process to gather information.

awead commented:
@cam156 so is it worth changing the Travis build? I'm not sure we're really slowing things down all the much with a single build as opposed to a split one.

nicktiberi committed:
Home page SEO improvements
- meta description added to home page
- "read more" link is now more descriptive

awead commented on app/views/layouts/frontend.html.erb:
```diff
@@ -8,6 +8,7 @@
     <!-- Internet Explorer use the highest version available -->
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <%= yield(:meta_tags) if content_for? :meta_tags %>
+    <%= render partial: 'resources/home_meta_tags' if current_page?('/') -%>
```
@nicktiberi On line 10, there's a yield, which is basically saying, if you have a content block that you've called `meta_tags`, I'm going to let it through right here. You can read up on on it if you like https://guides.rubyonrails.org/layouts_and_rendering.html#using-the-content-for-method

So, what you could do here is edit `app/views/pages/home.html.erb` and add something like:

``` ruby
<% content_for :meta_tags do %>
  <meta name="description" content="<%= t('home.intro.content') %>">
<% end %>
```

And that ought to do the trick. If this doesn't make sense, let me know and we can pair up on it.

nicktiberi replied:
Learning something new about Rails every day. 🤓 Thanks!

awead reviewed (approved):
👏

awead commented:
@nicktiberi could you verify with the lighthouse site that this fixes the problem?

nicktiberi commented:
@awead - yep, I verified with Chrome devtools that our SEO score is now 💯 :

<img width="338" alt="Screen Shot 2021-05-18 at 3 03 15 PM" src="https://user-images.githubusercontent.com/639920/118708814-312a0600-b7ea-11eb-91a9-45700e5ea125.png">
